packages/showcases/css/stories/components/navigation/tabs/examples/medium.html

```diff
@@ -1,110 +1,174 @@
 <div class="block">
-  <nav aria-label="Tabs list" class="vtmn-tabs vtmn-tabs_size--medium">
-    <ul>
-      <li><a href="" class="selected">Tab Name</a></li>
-      <li><a href="">Tab Name</a></li>
-      <li><a href="">Tab Name</a></li>
-      <li><a href="">Tab Name</a></li>
-    </ul>
-    <div class="vtmn-divider" role="separator"></div>
-  </nav>
+  <ul
+    class="vtmn-tabs vtmn-tabs_size--medium"
+    role="tablist"
+    aria-label="Medium tabs navigation"
+  >
+    <li>
+      <button role="tab" aria-controls="tabpanel-medium-1" aria-selected="true">
+        Tab Name 1
+      </button>
+    </li>
+    <li>
+      <button role="tab">Tab Name 2</button>
```

Comment:
Pure html showcase is a bit odd because none of the element here seems to be reactively/dynamically managed. I mean, in order to make this demo WCAG compliant, each button should have an `id` (to be paired with the `aria-labelledby` of the corresponding panel) and an `aria-selected` attribute defaulting to `false`. Not selected button should also have a `tabindex` set to `-1` so the next focusable element is the panel itself. Should we adapt the demo in this direction ? @GaspardMathon @lauthieb 

We could discuss that on videoconference if you want

Reply:
Agree!